fw/http_limits.c

```diff
@@ -1142,14 +1140,13 @@ frang_http_req_process(FrangAcc *ra, TfwConn *conn, TfwFsmData *data,
 
 	/* Ensure that HTTP request method is one of those defined by a user. */
 	T_FSM_STATE(Frang_Req_Hdr_Method) {
-		if (f_cfg->http_methods_mask) {
```

Comment:
I removed checking f_cfg->http_methods_mask  because this checking breaks new logic.